template.rb

```diff
@@ -45,14 +45,23 @@ def apply_template!(template_root)
   apply 'config/template.rb'
   apply '.gitignore.rb'
 
+  if ENV['CI']
```

Comment:
we don't need the ENV['CI'] anymore @noor-ahmed, you can check how another addon did that for example https://github.com/nimblehq/rails-templates/pull/232/files

Reply:
Yep, we'll remove it. We kept it for time being only.